reference/outlook/preview/simple-types.md

```diff
@@ -244,9 +261,64 @@ An array of `PhoneNumber` objects containing the phone numbers found in an email
 
 |Requirement| Value|
 |---|---|
-|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)| 1.0|
-|[Minimum permission level](../../../docs/outlook/understanding-outlook-add-in-permissions.md)| ReadItem|
-|Applicable Outlook mode| Read|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|1.0|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Read|
+
+
+#### RecurrenceProperties
+
+Gets or sets the properties of the recurrence.
+
+##### Type:
+
+* Object
+
+##### Properties:
+
+|Name|Type|Description|
+|---|---|---|
+|`interval`|Integer|Represents the period between recurrences of the same meeting or appointment series.|
+|`dayOfMonth`|Integer|Represents the day of the month.|
+|`dayOfWeek`|[Office.MailboxEnums.Days](Office.MailboxEnums.md#days-string)|Represents the day of the week or type of day, for example, weekend day vs weekday.|
+|`days`|Array of [Office.MailboxEnums.Days](Office.MailboxEnums.md#days-string)|Represents the set of days for this recurrence. Valid values are: `Mon`, `Tue`, `Wed`, `Thu`, `Fri`, `Sat`, and `Sun`.|
+|`weekNumber`|[Office.MailboxEnums.WeekNumber](Office.MailboxEnums.md#weeknumber-string)|Represents the number of the week in the selected month e.g. `first` for first week of the month.|
+|`month`|[Office.MailboxEnums.Month](Office.MailboxEnums.md#month-string)|Represent the month.|
+|`firstDayOfWeek`|[Office.MailboxEnums.Days](Office.MailboxEnums.md#days-string)|Represents your chosen first day of the week otherwise the default is the value in the current user's settings. Valid values are: `Mon`, `Tue`, `Wed`, `Thu`, `Fri`, `Sat`, and `Sun`.|
+
+##### Requirements
+
+|Requirement|Value|
+|---|---|
+|[Minimum mailbox requirement set version](../tutorial-api-requirement-sets.md)|Preview|
+|[Minimum permission level](https://docs.microsoft.com/outlook/add-ins/understanding-outlook-add-in-permissions)|ReadItem|
+|Applicable Outlook mode|Compose or read|
+
+
+#### RecurrenceTimeZone
+
+Represents the time zone applied to the recurrence.
+
+Type:
+
+* Object
+
+##### Properties:
+
+|Name| Type| Description|
+|---|---|---|
+|`name`| [Office.MailboxEnums.RecurrenceTimeZone](Office.MailboxEnums.md#recurrencetimezone-string) |Represents the Windows time zone. For example: "Pacific Standard Time" or Microsoft.Office.WebExtension.MailboxEnums.RecurrenceTimeZone.|
+|`offset`| Number |Integer value representing the number of minutes difference between the local time zone and UTC.|
```

Comment:
Do we want to say that only 'name' is recognized during setAsync? (i.e. setting the right name 'Pacific Standard Time' and the wrong offset (-10) would only take 'Pacific Standard Time') Also only setting the 'offset' without 'name' would be an error.

Reply:
so "offset" is optional?

Reply:
yes. If they just put the name, we will accept that. If they put the wrong offset and a name, we will make the offset whatever the offset should actually be according to the name.